presto-docs/src/main/sphinx/functions/math.rst

```diff
@@ -153,6 +153,23 @@ Mathematical Functions
     array ``bins``. The ``bins`` parameter must be an array of doubles and is
     assumed to be in sorted ascending order.
 
+Statistical Functions
+-----------------------
+
+.. function:: wilson_interval_lower(pos, neg[, z]) -> double
```

Comment:
The only reason I wrote them this way was to make them more efficient - I don't want to convert from a probability to a z-score in every function execution.

I can add a function that does that conversion.  Out of curiosity, will presto optimize a case like `wilson_interval_lower(pos, neg, z_score(0.95))` so that it only calls `z_score(0.95)` once rather than for each row?